src/aatoolbox/config/countries/cod.yaml

```diff
@@ -0,0 +1,7 @@
+iso3: cod
+codab:
+  hdx_address: cod-ab-cod
+  hdx_dataset_name: cod_admbnda_rgc_itos_20190911_SHP.zip
+  #the adm2 layer name is cod_admbnda_adm2_rgc_20190911.shp
```

Comment:
Does this mean that admin 2 doesn't follow the same structure as 0-1? I was afraid this might happen at some point. The way we could handle this for now is by adding admin 2 as a custom layer name, see the Nepal config as an example. Then add a comment in the config file.

Reply:
Oeh awesome! Like I did now?